cli/node/pep-get-enrollment-id.js

```diff
@@ -18,7 +18,7 @@
 'use strict';
 
 const program = require('commander');
-const pep_server = require('./../dispatcher-sdk-js');
+const pep_server = require('./../../api-wrapper-js');
```

Comment:
JS changes?

Thought this PR was bash-only

Reply:
Ya this should be removed from this PR.